packages/core/src/r4b/extensions.test.ts

```diff
@@ -285,8 +285,26 @@ describe("extensions", () => {
     patient.birthSex = "OTH";
     const anotherPatient = cloneResource(patient);
     expect(anotherPatient).not.toBe(patient);
-    expect(anotherPatient.birthSex).toEqual("OTH");
     expect(anotherPatient).toBeInstanceOf(CustomPatient);
+    expect(anotherPatient).toEqual(
+      expect.objectContaining({
+        birthSex: "OTH",
+        resourceType: "Patient",
+        toFhirResource: expect.any(Function),
+        toJSON: expect.any(Function),
+        computedName: expect.any(Function),
```

Comment:
Are these net new expectations? If so, why do we need them?

_Sorry, I can't think of how to ask that doesn't sound rude. I'm just curious about the justification for the change in test expectations. It's quite possible I don't fully understand the change._

Reply:
These are **not** net new expectations. They are expectations that were not previously being explicitly tested.

We expect a clone of an extended resource to act like the original object. This means having its custom properties and custom functions. 

When removing the Proxy and testing the cloning I found that the functions were not being copied so I added these tests.